### DISCUSSION TITLE
[MacBookPro Apple M1] Can't add new database

### DISCUSSION THREAD

**Discussion by Wayler** [pawelsalawa/sqlitestudio#4711] [Q&A]
### Details
Here you describe your problem details, or proposal for enhancement.

Hi

Can't add a new database on MacBookPro Apple M1. There is no option to do this in user interface.

<img width="950" alt="Zrzut ekranu 2023-03-2 o 11 20 15" src="https://user-images.githubusercontent.com/720719/222402780-73029883-0a1a-4f5b-aa73-e5cb890f2017.png">

### Steps to reproduce
This applies only to bug reports. If this is a feature request, delete this section.

### Operating system
Apple M1 Ventura 13.01

### SQLiteStudio version
This applies only to bug reports. If this is a feature request, delete this section.

**Comment by pawelsalawa** [comment]
What do you mean there is no option? The "OK" button is right there.

**Comment by Wayler** [comment]
Oh. I'm sorry. I go through tutorial with attached screen where is "plus" icon which I thought allow to add database.
![Add_DB2](https://user-images.githubusercontent.com/720719/222412084-91290b68-ad62-4e92-8aae-ea4b2578715d.jpg)
After your comment I dig deeper and I created new database with success.
